Misc/NEWS.d/next/Library/2023-04-27-14-50-03.gh-issue-103925.khC-El.rst

```diff
@@ -0,0 +1 @@
+_guess_quote_and_delimiter should be able to handle windows \r
```

Comment:
I believe for news entries, we normally state our changes, rather than the expected behavior. So, instead of "XX should be able to YY", you could say "Supported YY in XX". nit: `_guess_quote_and_delimiter` is an internal function that's not documented, even though it's totally fine to talk about that in news entry, your code actually solves the issue for `sniff` - that's clearer to users I think. Also you can quote the function with `` :func:`function_name` ``